{{ cookiecutter.project_slug }}/flows/query.yml

```diff
@@ -18,10 +15,8 @@ pods:
     separated_workspace: true
     polling: all
     uses_reducing: _merge_all
+    uses_after: pods/craft.yml
```

Comment:
I am not sure about the implication of this, but shouldn't it be `uses_before`? The `Sentencizer` and the `ImageReader/Normalizer` should happen before the encoding, not?

Reply:
Perhaps my understanding of `uses_before` and `uses_after` is also exactly wrong. My intuitive understanding is the following run order:
```
uses_before()
uses()
uses_after()
```
If you are sure it is 
```
uses_after()
uses()
uses_before()
```
then please ignore my comments. Otherwise, could you double-check?

Reply:
Taking inspiration from `https://github.com/jina-ai/jina/blob/master/jina/resources/helloworld.flow.query.yml` where `uses_after` is used

Reply:
`uses_before` and `uses_after` refer to what runs inside head pea and tail pea

Reply:
Do not `uses_before` and `uses_after` to concatenate executors

Reply:
`uses_before` and `uses_after` is about `head-pea` and `tail-pea`. If you want to concatenate sequentially executors in the same Pod or Pea, `CompoundExecutor` is the way to go

Reply:
retained `crafter` for both `index` and `query` flows.